SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/OutgoingMessages.h

```diff
@@ -100,14 +101,16 @@ struct FRemoveComponent : FOutgoingMessage
 
 struct FComponentUpdate : FOutgoingMessage
 {
-	FComponentUpdate(Worker_EntityId InEntityId, const Worker_ComponentUpdate& InComponentUpdate)
+	FComponentUpdate(Worker_EntityId InEntityId, const Worker_ComponentUpdate& InComponentUpdate, const TraceKey& InTrace)
```

Comment:
I don't think it's worth passing a reference to an integer. Integer is usually passed by value. Are there any reasons not to do it here? Could it be the TraceKey type becomes something more than int that we don't want to copy?

Reply:
Yep, this was my reasoning. I'm happy to change these const &s though.

Reply:
If there's an intention to make it a more complicated data, than int, I'm fine with keeping it const& then, resolving the comments.